packages/plugin-list/src/ListItemXmlDomParser.ts

```diff
@@ -25,9 +28,13 @@ export class ListItemXmlDomParser extends AbstractParser<Node> {
         }
         const nodes: VNode[] = [];
         let inlinesContainer: VNode;
-        // Parse the list item's attributes into a technical key of the node's
-        // attributes, that will be read only by ListItemDomRenderer.
-        const itemAttributes = { 'li-attributes': this.engine.parseAttributes(item) };
+        // Parse the list item's attributes into the node's ListItemAttributes,
+        // which will be read only by ListItemDomRenderer.
+        const attributes = this.engine.parseAttributes(item);
+        let itemAttributes;
+        if (attributes) {
+            itemAttributes = new ListItemAttributes(attributes);
+        }
```

Comment:
I think I would have done something like:
```typescript
const itemAttributes = this.engine.parseAttributes(item);
```
Then only later in the code:
```typescript
new ListItemAttributes(itemAttributes);
```
Since you `clone` it anway so in truth you don't need to create it here.